stream-chat-android-ui-common/src/test/kotlin/com/getstream/sdk/chat/utils/livedata/TestObserver.kt

```diff
@@ -0,0 +1,16 @@
+package com.getstream.sdk.chat.utils.livedata
+
+import androidx.lifecycle.Observer
+
+internal class TestObserver<T> : Observer<T> {
```

Comment:
This exists in three places now, can we move it to the `stream-chat-android-test` module?

Reply:
We still have two of these remaining, the extra one is called `LiveDataTestObserver` in the offline module.